README.md

```diff
@@ -648,6 +657,26 @@ repositories will only contain content that Cachito has made available.
 These repositories are created per request and deleted when the request is marked as stale or the
 request fails.
 
+### Nexus for RubyGems
+
+The RubyGems package manager functionality relies on [Nexus Repository Manager 3][nexus-docs] to
+store RubyGems dependencies. The Nexus instance consists of two repositories that act as a long
+terms storage - RubyGems proxy repository (e.g. `cachito-rubygems-proxy`) that points to
+`rubygems.org` and raw repository (e.g. `cachito-rubygems-raw`) used for storing Git dependencies.
+The RubyGems proxy repository caches all RubyGems packages that Cachito downloads through it and the
+raw repository holds tarballs of Git dependencies that Cachito uploads there after fetching them
+from the original locations.
+
+On each request, Cachito creates a RubyGems hosted repository (e.g. `cachito-rubygems-hosted-1`)
+and uploads there all GEM dependencies for the request. This repository is created per request and
+deleted when the request is marked as stale or the request fails. Redirecting Bundler to use this
+repository instead of a default RubyGems server is done by providing a configuration file. Note that
+there's no request specific repository for external dependencies as other package managers do.
```

Comment:
> there's no request specific repository for external dependencies

Might be worth mentioning how it's done instead? (Or just link to the relevant section below)